zh/guide/assets.md

```diff
@@ -18,19 +18,21 @@ description: 默认情况下 Nuxt 使用 vue-loader、file-loader 以及 url-loa
 ----| index.vue
 ```
 
-如果我们在CSS代码中使用 `url('~assets/image.png')`, 那么编译后它将被转换成 `require('~assets/image.png')`。
+如果我们在CSS代码中使用 `url('~assets/image.png')`, 那么编译后它将被转换成 `require('~/assets/image.png')`。
+
+> 请注意，由于css-loader升级，从Nuxt 2.0开始，你应该在CSS文件中使用`~assets`（没有斜杠），例如background:url("~assets/banner.svg")
 
 又或者如果我们在 `pages/index.vue` 中使用以下代码引用图片资源：
 ```html
 <template>
-  <img src="~assets/image.png">
+  <img src="~/assets/image.png">
```

Comment:
**i added `~/assets` But the Chinese official website shows that it is still wrong.**
@Atinux 
![tim 20181018115213](https://user-images.githubusercontent.com/28696399/47130442-4a821300-d2cc-11e8-8afe-51bc4dbb1b43.png)